modP2p/src/org/aion/p2p/Ver.java

```diff
@@ -8,16 +8,20 @@
 
     public static final short V0 = 0;
 
-    // for test
+    // new protocol version supporting fast sync
     public static final short V1 = 1;
 
+    // for test
+    public static final short VTEST = Byte.MAX_VALUE;
```

Comment:
Could also use a negative number ... might be slightly clearer when looking at the actual number from a log file or something

Reply:
My intention here was only to not have to update the test version with every newly introduced protocol. Using negative values may have other ramifications that are not worth the trouble, given the fact that there exists only a minor test for this. If this starts being used for more substantial testing, the value can be updated to something more meaningful at that point.